runtime/mixed.inl

```diff
@@ -1673,8 +1675,113 @@ inline int64_t operator>>(const mixed &lhs, const mixed &rhs) {
   return lhs.to_int() >> rhs.to_int();
 }
 
+template <typename Arg1, typename Arg2>
+struct conversion_php_warning_string {
+  static constexpr const char *text = "";
+};
+
+template<>
+struct conversion_php_warning_string<int64_t, const string &> {
```

Comment:
the const reference overloading doesn't look good, it is better to use a [decayed](https://en.cppreference.com/w/cpp/types/decay) (clear) type without cv and refs.

Reply:
Done.